### ISSUE TITLE
[Feature request] x-callback-url

### ISSUE THREAD

**Issue by greiggy** [standardnotes/forum#251]
Any possibility of being able to copy a x-callback-url link to access a note from another app e.g. a progress list?

Some apps expect you to write the x-callback-url link with the title, which is OK, but a neater way is to auto-generate the link and copy to clipboard.

I use these links to keep track of different parts of a project. Really helpful!

**Comment by mobitar** [comment]
Hmm that would be cool. We don't currently have support for deeplinking functionality. It's my goal to spend more quality time with the mobile apps, but currently still heavily focused on deep architectural improvements for the web and desktop app. There's a couple things I want to explore for mobile and deeplinking would be one of them.

**Comment by greiggy** [comment]
Looks like you're halfway there already.

Thanks for thinking about this. It is a cool feature!

--IanG


On 10 July 2018 at 17:21:22, Mo Bitar (notifications@github.com) wrote:

Hmm that would be cool. We don't currently have support for deeplinking
functionality. It's my goal to spend more quality time with the mobile
apps, but currently still heavily focused on deep architectural
improvements for the web and desktop app. There's a couple things I want to
explore for mobile and deeplinking would be one of them.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/standardnotes/forum/issues/251#issuecomment-403882651>,
or mute the thread

**Comment by tylermolamphy** [comment]
Sorry to resurrect an old conversation, but I’d like to add my interest in StandardNotes declaring a URL scheme in the iOS app. Specifically, I’d love the ability to create a new note from content passed via the URL string, so that I can bring content into StandardNotes and keep editing it from there. 

Currently I copy and paste content into a new note, and since I do this more than daily, I would love to do this automatically by calling a URL scheme.

I think this would be a great way to introduce light x-callback-url support while you work on the much larger issue of deeplinking in notes. Let me know what you think.

**Comment by mobitar** [comment]
cc @radko93

**Comment by radko93** [comment]
@tylermolamphy yes, sharing from other apps makes a lot of sense and it's on my TODO list.

**Comment by tylermolamphy** [comment]
That’s great to hear, thank you! I’ll be here if you’d like a tester.

**Comment by thed0ctor** [comment]
Another vote for this feature. Does anyone have a workaround in the meantime?

**Comment by JaspalSuri** [comment]
I'm currently not aware of any workarounds for it.

**Comment by JaspalSuri** [comment]
>From a 5-year supporter

Thanks for your support! 

>A shortcut to open the app via URL Scheme is useful to some users.

We agree that this feature would be useful and I've taken note of it for the dev team to review as they work on the next set of major releases.

**Comment by pazu88** [comment]
Great to hear it!

All the best,

Sent from ProtonMail Mobile

開啟 週二, 8月 25, 2020 在 14:15, Jaspal Suri <notifications@github.com> 寫道：
